R/otp-route.R

```diff
@@ -0,0 +1,287 @@
+#' Get the geometry of a route from the OTP
+#'
```

Comment:
@mem48 I think we probably need to make clear in the sf column names what the units are for duration, time, distance, elevationGained etc. E.g change labels to distance_km, time_seconds etc. Rather than relying on the user knowing?